packages/sql-common/README.md

```diff
@@ -1,6 +1,14 @@
 # Common Utils for OpenTelemetry SQL packages
 
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
 This is an internal utils package used for the different SQL instrumentations:
 
 1. mysql2
 2. pg
+
+[license-url]: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/LICENSE
```

Comment:
also not being used

Reply:
applied suggestion in https://github.com/open-telemetry/opentelemetry-js-contrib/pull/3098/commits/d457a7341f2cb4d19076f75cebb9451461af0ddc